R/get_generic.R

```diff
@@ -0,0 +1,72 @@
+## function name get_generic
+
+#' retrieve all parameters of specified type
+#' @param data parameter dataframe outputted from load_epidata()
+#' @param parameter_name name of the parameter type to retrieve, ensuring the name matches that in data
```

Comment:
We don't give the user any way of ensuring that the parameter name matches that in our tables. It would be good to have a think about how we can do this? So that the user could say get_parameter(params, "serial interval") and we would fetch them all things serial interval.